bench/release.py

```diff
@@ -4,34 +4,41 @@
 import sys
 import semantic_version
 import git
-import json
 import requests
 import getpass
-import argparse
 import re
 from requests.auth import HTTPBasicAuth
 import requests.exceptions
 from time import sleep
 from .config.common_site_config import get_config
 import click
 
+branches_to_update = {
+	'develop': [],
```

Comment:
settings develop key in branches_to_update in common_site_config. As bench release also used by other community users, if they don't maintain hotfix branch then it will break the release process